core/src/main/java/org/apache/iceberg/rest/RESTSessionCatalog.java

```diff
@@ -1206,12 +1006,15 @@ public void commitTransaction(SessionContext context, List<TableCommit> commits)
           UpdateTableRequest.create(commit.identifier(), commit.requirements(), commit.updates()));
     }
 
-    client.post(
-        paths.commitTransaction(),
-        new CommitTransactionRequest(tableChanges),
-        null,
-        headers(context),
-        ErrorHandlers.tableCommitHandler());
+    AuthSession authSession = authManager.contextualSession(context, catalogAuth);
```

Comment:
```suggestion
    AuthSession contextualSession = authManager.contextualSession(context, catalogAuth);
```

Reply:
this is just a nit, but it would be good to generally align with how the session is named across this entire class